WWDC/AppCoordinator+SessionActions.swift

```diff
@@ -91,6 +91,11 @@ extension AppCoordinator: SessionActionsViewControllerDelegate {
             }
         case .authorized:
             self.saveCalendarEvent(viewModel: viewModel, eventStore: eventStore)
+        @unknown default:
+            assertionFailure("An unexpected case was discovered on an non-frozen obj-c enum")
```

Comment:
Assertion failures so that the app doesn't crash, but instead takes a fail silently approach in production while stopping the debugger for development. Seems unnecessary to do the `fatalError()` approach for these enums, and I really would expect more enums to be frozen going forward.

Reply:
I agree